CLI/controllers/update_test.go

```diff
@@ -1,225 +1,204 @@
 package controllers_test
 
 import (
-	"cli/controllers"
 	"cli/models"
 	test_utils "cli/test"
-	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
-func TestUpdateTagColor(t *testing.T) {
```

Comment:
What happened to the original tests? Like TestUpdateTagColor.

Reply:
They were deleted with outdated interact tests... They're back in [97e859e](https://github.com/ditrit/OGrEE-Core/pull/493/commits/97e859e8c9a36429e4c75a5cc027eb9cee162ad7)